src/components/MovieCardContainer/MovieCardContainer.js

```diff
@@ -1,13 +1,56 @@
-import React from 'react';
-import MovieCard from '../MovieCard/MovieCard'
-import './MovieCardContainer.css'
-
-const MovieCardContainer = () => {
-  return (
-    <section className="movie-card-container">
-      <MovieCard />
-    </section>
-  )
+import React, { Component } from 'react';
+import MovieCard from '../MovieCard/MovieCard';
+import './MovieCardContainer.css';
+import { loadMovies } from '../../actions';
+import { connect } from 'react-redux';
+
+class MovieCardContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      movies: []
+    }
+  }
+
+  componentDidMount() {
+    console.log(this.props)
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v1/movies')
+      .then(response => response.json())
+      .then(movieList => {
+        this.props.loadMovies(movieList.movies)
+        this.setState({movies: movieList.movies})
+      })
+      .catch(err => console.error(err.message))
+  }
+
+  render() {
+    const movieCards = this.state.movies.map(movie => {
```

Comment:
@monstaro @lmanriq movies is going to be passed into this component from the global store as a prop. If you want those movies to be held in state instead, look into setting that state again in a [componentDidUpdate](https://reactjs.org/docs/react-component.html#componentdidupdate) or [getDerivedStateFromProps -- not always a necessary practice](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops) lifecycle method.

Otherwise, just replace `state` with `props` here and you might be good to go.

Reply:
@dreardon1021 @Trond240

Reply:
So thats the issue we have been having. doing this.props.movies.map returns 

"cant map undefined" So confused as what to do here